en/misc.ftl

```diff
@@ -111,7 +111,22 @@ trackerreport-faq-cta = See more FAQs about { -brand-name-firefox-relay }
 trackerreport-loading = Loading your tracker removal report…
 trackerreport-load-error = There was an error generating your tracker removal report. Please refresh the page to try again.
 
-## Countdown timer (e.g. to count down to pricing increases)
+## Tracker warning page
+
+contains-tracker-title = This link or image contains a tracker
+# Variables:
+#   $sender (string) - Sender of the email, e.g. person@email.com
+#   $datetime (string) - Time and date of when the email was sent, e.g. 05/06/2023 at 10:53pm EST
+contains-tracker-description =     Email trackers are surveillance tools that companies embed in emails to track your
```

Comment:
Is there a reason why there are white spaces before the first word of the string?